src/Listing.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import $ from 'jquery';
```

Comment:
No need for jquery inside of react. Instead of using $.get, we can use [fetch](https://developer.mozilla.org/en/docs/Web/API/Fetch_API#Concepts_and_usage).

See: [here](https://github.com/vandorjw/charity-auction-client/blob/testing/src/App.js) as an example.

@tkjone: what you think?